<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SIPl-DT-PS" failures="0" tests="2" name="Practice.Appium.ScrollDemo" time="20.162" errors="1" timestamp="2023-10-06T12:30:07 IST" skipped="1">
  <testcase classname="Practice.Appium.ScrollDemo" name="scrollIntoText" time="0.000">
    <skipped/>
  </testcase> <!-- scrollIntoText -->
  <system-out/>
  <testcase classname="Practice.Appium.ScrollDemo" name="configureAppium" time="20.162">
    <error message="The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Sipl\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1] 
Process output: [35m[Appium][39m Welcome to Appium v2.0.1 (REV HEAD)
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m&#039;127.0.0.1&#039;[39m }
[35m[Appium][39m Attempting to load driver uiautomator2...
[debug] [35m[Appium][39m Requiring driver at C:\Users\Sipl\.appium\node_modules\appium-uiautomator2-driver

" type="io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException">
      <![CDATA[io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: The local appium server has not been started. The given Node.js executable: C:\Program Files\nodejs\node.exe Arguments: [C:\Users\Sipl\AppData\Roaming\npm\node_modules\appium\build\lib\main.js, --port, 4723, --address, 127.0.0.1] 
Process output: [35m[Appium][39m Welcome to Appium v2.0.1 (REV HEAD)
[35m[Appium][39m Non-default server args:
[35m[Appium][39m { address: [32m'127.0.0.1'[39m }
[35m[Appium][39m Attempting to load driver uiautomator2...
[debug] [35m[Appium][39m Requiring driver at C:\Users\Sipl\.appium\node_modules\appium-uiautomator2-driver


at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:182)
at Practice.Appium.BaseTest.configureAppium(BaseTest.java:31)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:180)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.net.UrlChecker$TimeoutException: Timed out waiting for [http://127.0.0.1:4723/status] to be available after 20007 ms
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:94)
at io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:147)
at io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:169)
... 25 more
Caused by: java.util.concurrent.TimeoutException
at java.base/java.util.concurrent.FutureTask.get(FutureTask.java:204)
at org.openqa.selenium.net.UrlChecker.waitUntilAvailable(UrlChecker.java:91)
... 27 more
]]>
    </error>
  </testcase> <!-- configureAppium -->
  <system-out/>
</testsuite> <!-- Practice.Appium.ScrollDemo -->
